package-lock.json

```diff
@@ -3453,159 +3454,6 @@
         "url": "https://github.com/sponsors/gregberge"
       }
     },
-    "node_modules/@testing-library/dom": {
```

Comment:
Can you try running npm i and then pushing back up? not sure whats going on here with package-lock. Just want to be careful before merging it!

Reply:
Ok, I ran npm -i and pushed, let's see if these dependencies pass.